Modul_6_Dzhumelya/js/index.js

```diff
@@ -0,0 +1,90 @@
+// Вправа №1 Копія масива через метод concat()
+
+const arr = ['one', 10, 'name', 5, 'Nastya', 9];
+
+function arrayCopy () {
+    return arr.concat();
+}
+const newCopyArr = arrayCopy();
+console.log(newCopyArr);
+
+// Вправа №2 
+
+const arrTwo = ['The', 'quick', 'brown', 'fox', 'jumps', 'over', 'the', 'lazy', 'dog'];
+
+function arrayJoin (arr) {
+    return arr.join(' ');
+}
+const arrNewJoin = arrayJoin(arrTwo); 
+console.log('"' + arrNewJoin + '"');
+
+//  Вправа №3 
+
+const arrThree = [NaN, 0, 33, false, -51, '', undefined, 75, null];
+function removeFalsy (arr) {
+    return arr.filter((e) => e);
+}
+const arrNoFalsy = removeFalsy(arrThree);
+console.log(arrNoFalsy);
+
+//  Вправа №4 Два варіанта
+
+// Через цикл for
+function getWord () {
+    const arr1 = ['e', null, 'r'];
+    const arr2 = ['g', 'y'];
+    const arrUnite = [];
+    for (let i = 0; i< arr1.length; i++) {
+       arrUnite.push(arr1[i]);
+    }
+    for (let j = 0; j < arr2.length; j++) {
+        arrUnite.push(arr2[j]);
+    }
+    arrUnite.unshift('p', 'o');
+    arrUnite.push('m', 'o', 'd', 'e' );
+    arrUnite.splice(2, 0, 'w');
+    arrUnite.splice(6,2);
+    arrUnite[6] = 'c';
+    return arrayJoin(removeFalsy(arrUnite));
+}
+const arrNewUnite = getWord();
+console.log(arrNewUnite);
```

Comment:
Як опція, для виводу результатів можна обійтися без проміжної константи
```suggestion
console.log(getWord());
```